tests/conftest.py

```diff
@@ -33,13 +26,6 @@ def celery_config():
 
 
 @pytest.fixture(scope='module')
-def create_app(instance_path):
-    """Application factory fixture."""
-    def factory(**config):
-        app = Flask('testapp', instance_path=instance_path)
-        app.config.update(**config)
-        Babel(app)
-        InvenioRDMRecords(app)
-        app.register_blueprint(blueprint)
-        return app
-    return factory
+def create_app():
+    """Create app fixture."""
+    return create_api
```

Comment:
normal: This is the same `create_app` as `tests/api/conftest.py`. Should we have the following:

`tests/conftest.py`: `create_app` creates an api and ui application
`tests/api/conftest.py`: `create_app` creates an api only application
`tests/ui/conftest.py`: `create_app` creates ui only application

Reply:
:+1: Changed the ones. Missing UI for when we work on UI.